proposals/wgs/training/wgs.yaml

```diff
@@ -0,0 +1,85 @@
+wgs:
+- dir: wg-training
+  name: Training
+  mission_statement: >
+    WG Training covers developing, deploying, and operating training jobs on Kubeflow.
+  charter_link: charter.md
+  label: area/wg-training
+  leadership:
+    chairs:
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+    - github: terrytangyuan
+      name: Yuan Tang
+      company: Ant Group
+    tech_leads:
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Cisco
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: Jeffwan
+      name: Jiaxin Shan
+      company: Amazon Web Services
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+    - github: terrytangyuan
+      name: Yuan Tang
+      company: Ant Group
+  meetings:
+  - description: Regular WG Meeting (Asia & Europe friendly)
+    day: Wednesday
+    time: "03:00"
+    tz: PT (Pacific Time)
+    frequency: monthly - first Wednesday every month
```

Comment:
I would recommend a more frequent WG meeting while this is established.

Reply:
There is another meeting that's US friendly (see below). That would probably be sufficient. I would expect most of the communications done asynchronously on GitHub or Slack as members are from different time zones.

Reply:
@gaocegege I would also like to join the training working group, thanks!

Reply:
@ChanYiLin Thanks for your contribution. Added